### PR TITLE
[5366] - Snagging ticket for Bulk QTS

### PR DESCRIPTION
**Pull request by d-a-v-e**
### Context

A few minor content bugs were found and listed in https://trello.com/c/vpLYCAVu/5366-snagging-ticket-for-bulk-qts

### Changes proposed in this pull request

Addresses said content bugs

### CODE REVIEW

**Comment by d-a-v-e** [comment]
[[S] Snagging ticket for Bulk-QTS](https://trello.com/c/vpLYCAVu/5366-snagging-ticket-for-bulk-qts)

File: app/views/bulk_update/recommendations_uploads/_summary_bullets.html.erb [hunk 1]
```diff
@@ -5,8 +5,14 @@
...
-      <%= pluralize(awardable_rows_count, "trainee") %> who will be
-      recommended for <%= qts_or_eyts(recommendations_upload.awardable_rows) %>
+      <%= case awardable_rows_count
+        when 1
+          "The trainee"
+        when 2
+          "Both trainees"
+        else
+          "#{awardable_rows_count} trainees"
+        end %> who will be recommended for <%= qts_or_eyts(recommendations_upload.awardable_rows) %>
```
Comment (AbigailMcP) [inline app/views/bulk_update/recommendations_uploads/_summary_bullets.html.erb:15]: Ahh I think this is the wrong place. This content is fine as it was. The change needs to be made in `app/views/bulk_update/recommendations_uploads/_summary_all_dates.html.erb` i.e. when ALL trainees have been given dates and there are no errors.

Comment (AbigailMcP) [reply app/views/bulk_update/recommendations_uploads/_summary_bullets.html.erb:15]: Then in the first case:
```
The trainee will be recommended for QTS.
```

In the second case:
```
Both trainees will be recommended for QTS.
```

For anything else (no change):
```
All x trainees will be recommended for QTS.
```

**Review by AbigailMcP — APPROVED** [review]

**Comment by claire-hughez** [comment]
When 1 trainee is recommended for QTS, the upload summary page is missing a full stop.

'The trainee will be recommended for QTS' should be 'The trainee will be recommended for QTS.'

Same for when 2 trainees are recommended, it should have a full stop: 'Both trainees will be recommended for QTS'

And same when all trainees are recommended: 'All 30 trainees will be recommended for QTS or EYTS'

all other points on the ticket look good!

**Pull request merged**